src/AnnotationModeController.js

```diff
@@ -64,7 +64,12 @@ class AnnotationModeController extends EventEmitter {
      * @return {void}
      */
     registerThread(thread) {
-        this.threads.push(thread);
+        const page = thread.location.page || 1; // Defaults to page 1 if thread has no page'
```

Comment:
Do we want to make a change so that we default to null, in the case that we have an annotation that doesn't require pages? It might want to do it in another PR, though. Or do it now...

ie) video, audio, 3D models

Reply:
I'm ok with re-visiting this in another PR. I'll make a ticket for myself 👍